package.json

```diff
@@ -49,13 +49,17 @@
     "wait-until": "^0.0.2"
   },
   "dependencies": {
+    "@okta/okta-signin-widget": "^5.1.4",
+    "bootstrap": "^4.5.3",
     "broken-link-checker": "^0.7.8",
+    "dedent": "^0.7.0",
```

Comment:
I'm of the opinion that we should have as few dependencies as possible, and this one seems unnecessary since it's just allowing us to made code formatting slightly nicer.